sensor/kubernetes/listener/listener_impl.go

```diff
@@ -13,14 +13,21 @@ import (
 	"github.com/stackrox/rox/sensor/kubernetes/client"
 	"github.com/stackrox/rox/sensor/kubernetes/eventpipeline/component"
 	"github.com/stackrox/rox/sensor/kubernetes/listener/resources"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	// See https://groups.google.com/forum/#!topic/kubernetes-sig-api-machinery/PbSCXdLDno0
 	// Kubernetes scheduler no longer uses a resync period and it seems like its usage doesn't apply to us
-	noResyncPeriod              = 0
-	clusterOperatorResourceName = "clusteroperators"
-	clusterOperatorGroupVersion = "config.openshift.io/v1"
+	noResyncPeriod = 0
+
+	osConfigGroupVersion                = "config.openshift.io/v1"
+	osClusterOperatorsResourceName      = "clusteroperators"
+	osImageDigestMirrorSetsResourceName = "imagedigestmirrorsets"
+	osImageTagMirrorSetsResourceName    = "imagetagmirrorsets"
+
+	osOperatorAlphaGroupVersion              = "operator.openshift.io/v1alpha1"
+	osImageContentSourcePoliciesResourceName = "imagecontentsourcepolicies"
```

Comment:
just curious why these are not with the other OpenShift consts

Reply:
There were already two const's in the file, I was expanding on the existing usage.  Where/What other OpenShift consts are you referring to?  Happy to move these wherever it makes sense.

Was hoping to find these consts in the openshift api / go client libs but search came up empty.

Reply:
Ahh - I separated them because the `*ResourceName`'s are associated with the `GroupVersion` preceding it.

For example, if using the 'fully qualified' name to get these resources via `kubectl`, would do the following (spacing adjusted for comparison):

```
kubectl get      clusteroperators.config.openshift.io
kubectl get imagedigestmirrorsets.config.openshift.io
kubectl get    imagetagmirrorsets.config.openshift.io

   vs.

kubectl get imagecontentsourcepolicies.operator.openshift.io
```